virtcontainers/sandbox.go

```diff
@@ -1935,6 +1949,18 @@ func (s *Sandbox) calculateSandboxCPUs() uint32 {
 	return utils.CalculateVCpusFromMilliCpus(mCPU)
 }
 
+func (s *Sandbox) beforeHotplugHugeMem(currentMemSizeInMB uint32) error {
```

Comment:
Please can you add a comment explaining what this function does and why it is needed.